features-json/css-matches-pseudo.json

```diff
@@ -0,0 +1,268 @@
+      "title":"Mozilla Bug 906353 - Add support for css4 selector :matches(), the standard of :-moz-any()"
+    },
+    {
+      "url":"https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/9361350--matches",
+      "title":"Microsoft Edge UserVoice feature request for :matches()"
+    },
+    {
+      "url":"http://output.jsbin.com/lehina",
+      "title":"JS Bin testcase"
+    }
+  ],
+  "bugs":[
+    
+  ],
+  "categories":[
+    "CSS"
+  ],
+  "stats":{
+    "ie":{
+      "5.5":"n",
+      "6":"n",
+      "7":"n",
+      "8":"n",
+      "9":"n",
+      "10":"n",
+      "11":"n"
+    },
+    "edge":{
+      "12":"n",
+      "13":"u",
+      "14":"u"
+    },
+    "firefox":{
+      "2":"u",
+      "3":"u",
+      "3.5":"u",
+      "3.6":"u",
+      "4":"a x #3",
+      "5":"a x #3",
+      "6":"a x #3",
+      "7":"a x #3",
+      "8":"a x #3",
+      "9":"a x #3",
+      "10":"a x #3",
+      "11":"a x #3",
+      "12":"a x #3",
+      "13":"a x #3",
+      "14":"a x #3",
+      "15":"a x #3",
+      "16":"a x #3",
+      "17":"a x #3",
+      "18":"a x #3",
+      "19":"a x #3",
+      "20":"a x #3",
+      "21":"a x #3",
+      "22":"a x #3",
+      "23":"a x #3",
+      "24":"a x #3",
+      "25":"a x #3",
+      "26":"a x #3",
+      "27":"a x #3",
+      "28":"a x #3",
+      "29":"a x #3",
+      "30":"a x #3",
+      "31":"a x #3",
+      "32":"a x #3",
+      "33":"a x #3",
+      "34":"a x #3",
+      "35":"a x #3",
+      "36":"a x #3",
+      "37":"a x #3",
+      "38":"a x #3",
+      "39":"a x #3",
+      "40":"a x #3",
+      "41":"a x #3",
+      "42":"a x #3",
+      "43":"a x #3",
+      "44":"a x #3",
+      "45":"a x #3",
+      "46":"a x #3",
+      "47":"a x #3"
+    },
+    "chrome":{
+      "4":"u",
+      "5":"u",
+      "6":"u",
+      "7":"u",
+      "8":"u",
+      "9":"u",
+      "10":"u",
+      "11":"u",
+      "12":"u",
+      "13":"u",
+      "14":"u",
+      "15":"u",
+      "16":"u",
+      "17":"u",
+      "18":"u",
+      "19":"u",
+      "20":"u",
+      "21":"u",
+      "22":"u",
+      "23":"u",
+      "24":"u",
+      "25":"u",
+      "26":"a x #1",
+      "27":"a x #1",
+      "28":"a x #1",
+      "29":"a x #1",
+      "30":"a x #1",
+      "31":"a x #1",
+      "32":"a x #1",
+      "33":"a x #1",
+      "34":"a x #1",
+      "35":"a x #1",
+      "36":"a x #1",
+      "37":"a x #1",
+      "38":"a x #1",
+      "39":"a x #1",
+      "40":"a x #1",
+      "41":"a x #1",
+      "42":"a x #1",
+      "43":"a x #1",
+      "44":"a x #1",
+      "45":"a x #1",
+      "46":"a x #1",
+      "47":"a x #1",
+      "48":"a x #1",
+      "49":"a x #1",
+      "50":"a x #1",
+      "51":"a x #1"
+    },
+    "safari":{
+      "3.1":"u",
+      "3.2":"u",
+      "4":"u",
+      "5":"u",
+      "5.1":"a x #1",
+      "6":"a x #1",
+      "6.1":"a x #1",
+      "7":"a x #1",
+      "7.1":"a x #1",
+      "8":"a x #1",
+      "9":"y #2",
+      "9.1":"y #2",
```

Comment:
@cvrebert FYI: Trailing comma here and at "android", causing invalid JSON. Not sure why the validator didn't catch it...maybe 'cause they're new files? Anyway, fixed it now.